index.html

```diff
@@ -669,8 +673,41 @@ <h3>Social</h3>
           </div>
         </div>
       </section>
+
+      <!-- organizers -->
+      <section id="organizers" class="wrapper style3 fade-up">
+        <div class="inner">
+          <h2>Organizers</h2>
+          <p>
+            Huge shout out to all the organizers that made this event happen!
```

Comment:
Putting myself in a participants' shoes, this comes off a little self-congratulatory.  Maybe something a bit more neutral like:

> Your team of volunteers who are working hard to make this event happen!

As Marjorie is a copywriter, she could be a good person to reach out to if we're unsure.

Reply:
I agree. Will use your suggestion for now, we can also ask Marjorie once she's available to chat.